src/indent.cpp

```diff
@@ -618,35 +618,25 @@ static void _log_indent_tmp(const char *func, const uint32_t line, parse_frame_t
 void indent_text(void)
 {
    LOG_FUNC_ENTRY();
-   chunk_t       *pc;
-   chunk_t       *next;
-   chunk_t       *prev       = nullptr;
-   bool          did_newline = true;
-   int           idx;
-   size_t        vardefcol    = 0;
-   size_t        shiftcontcol = 0;
-   size_t        indent_size  = cpd.settings[UO_indent_columns].u;
-   parse_frame_t frm;
-   bool          in_preproc          = false;
-   size_t        indent_column       = 0;
-   size_t        parent_token_indent = 0;
-   int           xml_indent          = 0;
-   bool          token_used;
-   size_t        sql_col      = 0;
-   size_t        sql_orig_col = 0;
-   bool          in_func_def  = false;
-   c_token_t     memtype;
-
-   memset(&frm, 0, sizeof(frm));
+   bool         did_newline   = true;
+   size_t       vardefcol     = 0;
+   const size_t indent_size   = cpd.settings[UO_indent_columns].u;
+   size_t       indent_column = 0;
+   int          xml_indent    = 0;
+   size_t       sql_col       = 0;
+   size_t       sql_orig_col  = 0;
+   bool         in_func_def   = false;
+
    cpd.frame_count = 0;
 
    // dummy top-level entry
+   parse_frame_t frm{};
    frm.pse[0].indent     = 1;
    frm.pse[0].indent_tmp = 1;
    frm.pse[0].indent_tab = 1;
    frm.pse[0].type       = CT_EOF;
 
-   pc = chunk_get_head();
+   auto pc = chunk_get_head();
```

Comment:
I think there is no need to force auto against simple types. In such cases, "auto" has no advantages, but it makes it difficult to read and refactor the code.